core/__tests__/actions/schedules.ts

```diff
@@ -80,6 +119,26 @@ describe("actions/schedules", () => {
             "schedules:list",
             connection
           );
+
+        expect(error).toBeUndefined();
+        expect(schedules.length).toBe(2);
+        expect(schedules[0].name).toBe("test schedule 2");
+        expect(total).toBe(2);
+      });
+
+      test("an administrator can list all schedules by model id", async () => {
+        connection.params = {
+          csrfToken,
+          modelId: "mod_profiles",
+        };
+        const { error, schedules, total } =
+          await specHelper.runAction<SchedulesList>(
+            "schedules:list",
+            connection
+          );
+
+        console.log(JSON.stringify(error));
```

Comment:
✏️

Reply:
Ah, I committed it but missed a push.  Thanks!